source/about/mattermost-v10-changelog.md

```diff
@@ -7,6 +7,81 @@ Support for Mattermost Server v9.5 [Extended Support Release](https://docs.matte
 - See the [changelog in progress](https://bit.ly/2nK3cVf) for details about the upcoming release.
 ```
 
+(release-v10.5-extended-support-release)=
+## Release v10.5 - [Extended Support Release](https://docs.mattermost.com/about/release-policy.html#release-types)
+
+**Release Day: Februrary 14, 2025**
+
+### Compatibility
+ - Updated minimum Safari version to 17.4+ and minimum Firefox version to 119+.
+
+### Upgrade
+ - v10.5 introduces Property System Architecture schema migration. See the [Important Upgrade Notes](https://docs.mattermost.com/upgrade/important-upgrade-notes.html) for details.
+ - The internal workings of the ``PluginLinkComponent`` in the webapp have been improved. Plugins that register link tooltips using ``registerLinkTooltipComponent`` will see changes in how tooltip components are managed. Link tooltips are now unmounted from the DOM by default, significantly improving performance. They are only mounted when a link is hovered over or focused. As a result, plugins need to update their components to properly handle mounting and unmounting scenarios.
```

Comment:
Should this be communicated as a breaking change? There may be no in-use third party plugins affected, but if there are, this is a breaking change for them.